### PR TITLE
add comment properties

### PR DESCRIPTION
**Pull request by gideonnnalue**
Desc:
Added comment features

### CODE REVIEW

File: .firebase/hosting.YnVpbGQ.cache [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+static/media/sendIcon.c7735e6b.svg,1576889708732,3fc33f4dbaec5ad868f53e46ff9481c53dd7883682092907ed38deb4d4b734f2
+static/css/2.7bce467f.chunk.css.map,1576889708822,7136d7aad6e3a747a5c4230597d30fa5e52c3a21813c8335a9def35726d2fc19
+static/media/defaultImg.b6681619.png,1576889708765,f622aa4adcb35325ed22b2b223aef5b8d27b0f2f7f8c76209563c48b91d414dc
+static/js/main.bbe4f31f.chunk.js.map,1576889708823,ef8dd5b2567510fd6cd72cd8ec47111bd3ae3466793b40c11d1328b287c035c6
+static/media/logo.1156e2ba.png,1576889708765,28fe321563e91546194a9e759ac28bb92184ec9af71809d9c78dcfa628be4226
+static/media/world-img.4d6f0bbb.png,1576889708795,dcc8ec425bfe22f31f0f23453e856e1444f690ba1f619c586ea5320289fb9778
+static/media/checkup.c5d870be.png,1576889708809,cda4dcf835739383b3f187e597ace6abbc5c20813b4b9f67151540b4037fa092
+static/js/2.9ae98fff.chunk.js,1576889708817,3d9d4fd63c6beb14b553f6186f68338e1be086b1fd2da376960cf149c372a77b
+static/media/postImg1.7f5a99ce.png,1576889708797,984d3ec3be81c3faaa3e6e94f9b45a35a83e17a22393ca84d54d1905b60bed0f
+static/js/2.9ae98fff.chunk.js.map,1576889708824,8c1be0b5555c7f15be67405635182e3d00eb669f25655b390d006a92748bd6c8
```
Comment (enyeInc) [inline .firebase/hosting.YnVpbGQ.cache]: why is this file being committed?

File: src/components/feed/reducer.js [hunk 2]
```diff
@@ -61,6 +62,20 @@ export default (state = INITIAL_STATE, action) => {
...
+				...state,
+				allPosts: state.allPosts.map((post, i) =>
+					post.id === action.payload.id
+						? {
+								...post,
+								commentCount:
+									state.allPosts[i].commentCount + 1,
+						  }
+						: post,
+				),
```
Comment (enyeInc) [inline src/components/feed/reducer.js]: wrapper this logic in a function, its a bit confusing the way that it is laid out

File: src/components/feed/services.js [hunk 3]
```diff
@@ -18,3 +18,9 @@ export const postLike = async (id) => {
 	const res = await Axios.get(`${POST_URL}/${id}/like`);
 	return res;
 };
+
+export const postComment = async ({ id, body }) => {
+	console.log(id);
```
Comment (enyeInc) [inline src/components/feed/services.js]: remove console log

File: src/components/feed/services.js [hunk 4]
```diff
@@ -18,3 +18,9 @@ export const postLike = async (id) => {
...
+	const res = await Axios.post(`${POST_URL}/${id}/comment`, { body: body });
+	return res;
+};
```
Comment (enyeInc) [inline src/components/feed/services.js]: just return the Axios post call directly

File: src/components/signin/components/Signin.jsx [hunk 5]
```diff
@@ -16,7 +18,7 @@ const { signinText } = signInData;
 const handleSubmit = (e, userData, dispatch, history) => {
     e.preventDefault();
     const userInfo = {};
-    Object.keys(userData).forEach(el => {
+    Object.keys(userData).forEach((el) => {
```
Comment (enyeInc) [inline src/components/signin/components/Signin.jsx]: revert this change

File: src/components/signin/components/Signin.jsx [hunk 6]
```diff
@@ -48,7 +50,7 @@ const renderInput = (formData, handleInputChange, error = false) => (
  *
  * @function {*} renderButton
  */
-const renderButton = state => (
+const renderButton = (state) => (
```
Comment (enyeInc) [inline src/components/signin/components/Signin.jsx]: revert this change

File: src/components/signin/components/Signin.jsx [hunk 7]
```diff
@@ -73,14 +75,22 @@ const Signin = () => {
     const { formData, handleInputChange } = useInputFields();
     const dispatch = useDispatch();
     const { history } = useReactRouter();
-    const signinState = useSelector(state => state.signin);
+    const signinState = useSelector((state) => state.signin);
```
Comment (enyeInc) [inline src/components/signin/components/Signin.jsx]: here as well

File: src/components/signin/components/Signin.jsx [hunk 8]
```diff
@@ -73,14 +75,22 @@ const Signin = () => {
...
     return (
         <AuthLayout>
             <div className="p-signinPage">
+                <div className="p-authPage__switch">
+                    Not yet a user?{' '}
```
Comment (enyeInc) [inline src/components/signin/components/Signin.jsx]: string in constants.js

File: src/components/signin/components/Signin.jsx [hunk 9]
```diff
@@ -73,14 +75,22 @@ const Signin = () => {
...
+                    <Link to="/signup" className="">
+                        Sign up
```
Comment (enyeInc) [inline src/components/signin/components/Signin.jsx]: here as well

File: src/components/signup/components/Signup.jsx [hunk 10]
```diff
@@ -16,7 +18,7 @@ const { signupText } = signUpData;
 const handleSubmit = (e, userData, dispatch, checked, history) => {
     e.preventDefault();
     const userInfo = {};
-    Object.keys(userData).forEach(el => {
+    Object.keys(userData).forEach((el) => {
```
Comment (enyeInc) [inline src/components/signup/components/Signup.jsx]: revert

File: src/components/signup/components/Signup.jsx [hunk 11]
```diff
@@ -68,7 +70,7 @@ const renderCheckedLabel = (checked, handleChecked) => (
  *
  * @function {*} renderButton
  */
-const renderButton = state => (
+const renderButton = (state) => (
```
Comment (enyeInc) [inline src/components/signup/components/Signup.jsx]: revert

File: src/components/signup/components/Signup.jsx [hunk 12]
```diff
@@ -94,14 +96,20 @@ const Signup = () => {
...
+                    Already a member?{' '}
+                    <Link to="/signin" className="">
+                        Sign in
+                    </Link>
+                </div>
                 <div className="p-signupPage__heading">{signupText}</div>
                 <form
                     className="a-formAuth"
-                    onSubmit={e =>
+                    onSubmit={(e) =>
```
Comment (enyeInc) [inline src/components/signup/components/Signup.jsx]: revert

File: src/components/signup/components/Signup.jsx [hunk 13]
```diff
@@ -94,14 +96,20 @@ const Signup = () => {
     const { checked, handleChecked } = useCheckedLabel();
     const dispatch = useDispatch();
     const { history } = useReactRouter();
-    const signupState = useSelector(state => state.signup);
+    const signupState = useSelector((state) => state.signup);
```
Comment (enyeInc) [inline src/components/signup/components/Signup.jsx]: revert

File: src/components/signin/components/Signin.jsx (hunk 9 above)
Comment (gideonnnalue) [reply src/components/signin/components/Signin.jsx]: All of these were as a result of an extension I think, or a setting. Will revert all
